searchapp/app/adapters/application.js

```diff
@@ -2,6 +2,7 @@ import DS from 'ember-data';
 
 export default DS.JSONAPIAdapter.extend({
   host: '//api.' + window.location.host,
+  host: 'http://api.lvh.me:5000',
```

Comment:
Thanks, switching all this back. Developing with ember-cli in the /searchapp route messes with the current routing configuration, so I have to swap some things around with the API.